src/code-view/styles.scss

```diff
@@ -44,12 +44,14 @@ $color-background-code-view-dark: #282c34;
 }
 
 .line-number {
-  border-inline-end-color: cs.$color-border-divider-default;
   vertical-align: text-top;
+  text-align: right;
```

Comment:
this seems a slightly unrelated change: are we sure it's needed? should it be `end` for RTL support? (although not sure what RTL looks like in a code editor context tbh)

Reply:
yeah agree it makes sense in general, addressing as a separate followup sounds good